spring-ai-core/src/main/java/org/springframework/ai/chat/client/advisor/observation/DefaultAdvisorObservationConvention.java

```diff
@@ -27,18 +28,9 @@
  * @author Christian Tzolov
  * @since 1.0.0
  */
-
 public class DefaultAdvisorObservationConvention implements AdvisorObservationConvention {
 
-	public static final String DEFAULT_NAME = "spring.ai.chat.client.advisor";
-
-	private static final String CHAT_CLIENT_ADVISOR_SPRING_AI_KIND = "chat_client_advisor";
-
-	private static final KeyValue ADVISOR_TYPE_NONE = KeyValue.of(LowCardinalityKeyNames.ADVISOR_TYPE,
-			KeyValue.NONE_VALUE);
-
-	private static final KeyValue ADVISOR_NAME_NONE = KeyValue.of(HighCardinalityKeyNames.ADVISOR_NAME,
-			KeyValue.NONE_VALUE);
+	public static final String DEFAULT_NAME = "spring.ai.advisor";
```

Comment:
@tzolov this name is only used for metrics. Since Micrometer generates several timer-related metrics with long names, I thought of making this base name shorter as well.

So, instead of having metrics with name "spring_ai_chat_client_advisor_active_milliseconds_count" we would have "spring_ai_advisor_active_milliseconds_count".

Thoughts? Since Advisors are not used anywhere else, we might not need the "chat.client" part in the base name.